### ISSUE TITLE
Project directory pkgs/ is not empty.

### ISSUE THREAD

**Issue by JGrubb** [Litipk/Jupyter-PHP-Installer#5]
```
~/Downloads [ php ./jupyter-php-installer.phar install -vvv                                                                                        ] 10:18 AM
You are running PHP-CLI with xdebug enabled. This will have a major impact on the kernel's performance.

You are running composer with xdebug enabled. This has a major impact on runtime performance. See https://getcomposer.org/xdebug
Installing litipk/jupyter-php (0.1)


  [InvalidArgumentException]
  Project directory pkgs/ is not empty.
```

Hey thanks for starting this project.  Can you point me in the right direction here?   `~/Library/jupyter-php` does not exist yet, so I'm not sure where to look for the pkgs dir that needs to be empty.

**Comment by castarco** [comment]
Hi @JGrubb , maybe are you executing the installer as root?

In any case, I've updated the installer to avoid conflicts with previous installations, if we're lucky this will solve your problems :) .

P.D.: I'm not a current OSX user, I'll be grateful for any insight about how to improve OSX support ^^.

**Comment by JGrubb** [comment]
Not running it as root, but I did try and install the old iPython PHP kernel.  Never had any success with it, but it's possible that some old files were left behind I'm just not sure where.

I will definitely help any way I can from the docs standpoint.  I'll try the new installer and let you know how it works (and will dig deeper if it doesn't).

Thank you!

**Comment by castarco** [comment]
Hi again @JGrubb , did the last change in the installer solve your problem?

Thx!

**Comment by JGrubb** [comment]
Sort of.  It now prints out the path to the non-empty directory, so after a few attempts (and installing php-zmq) I got it all installed.  It still took some fiddling - emptying out that directory, but making sure it's there in the first place.

I started up Jupyter notebook at that point but didn't have an option to start a PHP notebook.  And then I got pulled into a meeting :(

I should have some time today to actually dig into the source and see if I can sort it out.  I'd love to have this working...
